pyproject.toml

```diff
@@ -21,6 +21,7 @@ pucoti = "pucoti:app"
 python = "^3.12,<3.13"
 pygame-ce = "2.4.1"
 typer = "^0.12.3"
+pyobjc-framework-Cocoa = "10.3.1"
```

Comment:
I confirm that this doesn't work on linux, failing with `PyObjC requires macOS to build`. You should add the dependency only on macOS, using https://python-poetry.org/docs/dependency-specification/#using-environment-markers and having a try: except ModuleNotFoundError: to gate your logic in os_specific.py